### ISSUE TITLE
local_exploit_suggester -> NameError undefined local variable or method `rhost'

### ISSUE THREAD

**Issue by CaraTortu** [rapid7/metasploit-framework#14649]
I ran the metasploit module "post/multi/recon/local_exploit_suggester" and instead of outputting the local vulnerabilities, it outputted this:

[*] 192.168.1.140 - Collecting local exploits for x64/linux...
[*] 192.168.1.140 - 39 exploit checks are being tried...
[+] 192.168.1.140 - exploit/linux/local/network_manager_vpnc_username_priv_esc: The service is running, but could not be validated.
[+] 192.168.1.140 - exploit/linux/local/ptrace_sudo_token_priv_esc: The service is running, but could not be validated.
[-] 192.168.1.140 - Post failed: NameError undefined local variable or method `rhost' for #<Msf::Modules::Post__Multi__Recon__Local_exploit_suggester::MetasploitModule:0x00007f06215f64e0>
[-] 192.168.1.140 - Call stack:
[-] 192.168.1.140 -   /usr/share/metasploit-framework/modules/post/multi/recon/local_exploit_suggester.rb:153:in `run'
[*] Post module execution completed

I am currently in Kali Linux. Everything is updated and I tried to reinstall Metasploit by the apt repository. Nothing fixed this error so far. (Ruby is also last version)

**Comment by bcoles** [comment]
If you `set verbose true` and re-run the module it will print the name of each module it tries. This will help determine which module is breaking.

**Comment by bcoles** [comment]
Confirmed. Looks like there might be a couple of bugs here.

```
msf6 > 
[*] Sending stage (3008420 bytes) to 172.16.191.170
[*] Meterpreter session 1 opened (172.16.191.165:1337 -> 172.16.191.170:55798) at 2021-01-22 16:38:15 -0500

msf6 > use post/multi/recon/local_exploit_suggester 
msf6 post(multi/recon/local_exploit_suggester) > set session 1
session => 1
msf6 post(multi/recon/local_exploit_suggester) > set verbose true
verbose => true
rmsf6 post(multi/recon/local_exploit_suggester) > run

[*] 172.16.191.170 - Collecting local exploits for x64/linux...
[*] 172.16.191.170 - The following 41 exploit checks are being tried:
[*] 172.16.191.170 - exploit/linux/local/abrt_raceabrt_priv_esc
[*] 172.16.191.170 - exploit/linux/local/abrt_sosreport_priv_esc
[*] 172.16.191.170 - exploit/linux/local/af_packet_chocobo_root_priv_esc
[*] 172.16.191.170 - exploit/linux/local/af_packet_packet_set_ring_priv_esc
[*] 172.16.191.170 - exploit/linux/local/apport_abrt_chroot_priv_esc
[*] 172.16.191.170 - exploit/linux/local/asan_suid_executable_priv_esc
[*] 172.16.191.170 - exploit/linux/local/blueman_set_dhcp_handler_dbus_priv_esc
[*] 172.16.191.170 - exploit/linux/local/bpf_priv_esc
[*] 172.16.191.170 - exploit/linux/local/bpf_sign_extension_priv_esc
[*] 172.16.191.170 - exploit/linux/local/diamorphine_rootkit_signal_priv_esc
[*] 172.16.191.170 - exploit/linux/local/docker_daemon_privilege_escalation
[*] 172.16.191.170 - exploit/linux/local/docker_privileged_container_escape
[*] 172.16.191.170 - exploit/linux/local/exim4_deliver_message_priv_esc
[*] 172.16.191.170 - exploit/linux/local/glibc_ld_audit_dso_load_priv_esc
[*] 172.16.191.170 - exploit/linux/local/glibc_origin_expansion_priv_esc
[*] 172.16.191.170 - exploit/linux/local/glibc_realpath_priv_esc
[*] 172.16.191.170 - exploit/linux/local/gnu_mailutils_maidag_url_priv_esc
[*] 172.16.191.170 - exploit/linux/local/hp_xglance_priv_esc
[*] 172.16.191.170 - exploit/linux/local/juju_run_agent_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ktsuss_suid_priv_esc
[*] 172.16.191.170 - exploit/linux/local/lastore_daemon_dbus_priv_esc
[*] 172.16.191.170 - exploit/linux/local/libuser_roothelper_priv_esc
[*] 172.16.191.170 - exploit/linux/local/nested_namespace_idmap_limit_priv_esc
[*] 172.16.191.170 - exploit/linux/local/netfilter_priv_esc_ipv4
[*] 172.16.191.170 - exploit/linux/local/network_manager_vpnc_username_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ntfs3g_priv_esc
[*] 172.16.191.170 - exploit/linux/local/omniresolve_suid_priv_esc
[*] 172.16.191.170 - exploit/linux/local/overlayfs_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ptrace_sudo_token_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ptrace_traceme_pkexec_helper
[*] 172.16.191.170 - exploit/linux/local/rds_atomic_free_op_null_pointer_deref_priv_esc
[*] 172.16.191.170 - exploit/linux/local/rds_rds_page_copy_user_priv_esc
[*] 172.16.191.170 - exploit/linux/local/recvmmsg_priv_esc
[*] 172.16.191.170 - exploit/linux/local/reptile_rootkit_reptile_cmd_priv_esc
[*] 172.16.191.170 - exploit/linux/local/servu_ftp_server_prepareinstallation_priv_esc
[*] 172.16.191.170 - exploit/linux/local/systemtap_modprobe_options_priv_esc
[*] 172.16.191.170 - exploit/linux/local/test
[*] 172.16.191.170 - exploit/linux/local/ufo_privilege_escalation
[*] 172.16.191.170 - exploit/linux/local/vmware_alsa_config
[*] 172.16.191.170 - exploit/multi/local/magnicomp_sysinfo_mcsiwrapper_priv_esc
[*] 172.16.191.170 - exploit/multi/local/xorg_x11_suid_server_modulepath
[*] 172.16.191.170 - exploit/linux/local/abrt_raceabrt_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/abrt_sosreport_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/af_packet_chocobo_root_priv_esc: The target is not exploitable. Linux kernel 4.15.0-20-generic #21-Ubuntu is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/af_packet_packet_set_ring_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/apport_abrt_chroot_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/asan_suid_executable_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/blueman_set_dhcp_handler_dbus_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/bpf_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/bpf_sign_extension_priv_esc: The target is not exploitable. Kernel version 4.15.0-20-generic is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/diamorphine_rootkit_signal_priv_esc: The target is not exploitable. Diamorphine is not installed, or incorrect signal '64'
[*] 172.16.191.170 - exploit/linux/local/docker_daemon_privilege_escalation: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/docker_privileged_container_escape: The target is not exploitable. Not inside a Docker container
[*] 172.16.191.170 - exploit/linux/local/exim4_deliver_message_priv_esc: Cannot reliably check exploitability.
[*] 172.16.191.170 - exploit/linux/local/glibc_ld_audit_dso_load_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/glibc_origin_expansion_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/glibc_realpath_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/gnu_mailutils_maidag_url_priv_esc: The target is not exploitable. /usr/local/sbin/maidag is not set-uid
[*] 172.16.191.170 - exploit/linux/local/hp_xglance_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/juju_run_agent_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/ktsuss_suid_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/lastore_daemon_dbus_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/libuser_roothelper_priv_esc: The target is not exploitable.
[+] 172.16.191.170 - exploit/linux/local/nested_namespace_idmap_limit_priv_esc: The target appears to be vulnerable.
[*] 172.16.191.170 - exploit/linux/local/netfilter_priv_esc_ipv4: The target is not exploitable.
[+] 172.16.191.170 - exploit/linux/local/network_manager_vpnc_username_priv_esc: The service is running, but could not be validated.
[*] 172.16.191.170 - exploit/linux/local/ntfs3g_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/omniresolve_suid_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/overlayfs_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/ptrace_sudo_token_priv_esc: The target is not exploitable.
[+] 172.16.191.170 - exploit/linux/local/ptrace_traceme_pkexec_helper: The target appears to be vulnerable.
[*] 172.16.191.170 - exploit/linux/local/rds_atomic_free_op_null_pointer_deref_priv_esc: The target is not exploitable. Linux kernel 4.15.0-20-generic #21-Ubuntu is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/rds_rds_page_copy_user_priv_esc: The target is not exploitable. Linux kernel version 4.15.0-20-generic is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/recvmmsg_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/reptile_rootkit_reptile_cmd_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/servu_ftp_server_prepareinstallation_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/systemtap_modprobe_options_priv_esc: The target is not exploitable.
/bin/su
/bin/ping
/bin/fusermount
/bin/ntfs-3g
/bin/mount
/bin/umount
/usr/lib/openssh/ssh-keysign
/usr/lib/eject/dmcrypt-get-device
/usr/lib/xorg/Xorg.wrap
/usr/lib/policykit-1/polkit-agent-helper-1
/usr/bin/gpasswd
/usr/bin/arping
/usr/bin/chfn
/usr/bin/vmware-user-suid-wrapper
/usr/bin/passwd
/usr/bin/traceroute6.iputils
/usr/bin/chsh
/usr/bin/newgrp
/usr/bin/bwrap
/usr/bin/newuidmap
/usr/bin/sudo
/usr/bin/newgidmap
/usr/bin/pkexec
/sbin/mount.ecryptfs_private
/sbin/mount.cifs
[-] 172.16.191.170 - exploit/linux/local/test: Check failed
[*] 172.16.191.170 - exploit/linux/local/ufo_privilege_escalation: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/vmware_alsa_config: The target is not exploitable.
[*] 172.16.191.170 - exploit/multi/local/magnicomp_sysinfo_mcsiwrapper_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/multi/local/xorg_x11_suid_server_modulepath: The target is not exploitable.
[-] 172.16.191.170 - Post failed: NameError undefined local variable or method `rhost' for #<Msf::Modules::Post__Multi__Recon__Local_exploit_suggester::MetasploitModule:0x00005583162124a8>
[-] 172.16.191.170 - Call stack:
[-] 172.16.191.170 -   /root/Desktop/metasploit-framework/modules/post/multi/recon/local_exploit_suggester.rb:153:in `run'
[*] Post module execution completed
msf6 post(multi/recon/local_exploit_suggester) > 

```

**Comment by bcoles** [comment]
It looks like this is an issue with storing notes in the database.

```
msf6 post(multi/recon/local_exploit_suggester) > run

[*] 172.16.191.170 - Collecting local exploits for x64/linux...
[*] 172.16.191.170 - The following 41 exploit checks are being tried:
[*] 172.16.191.170 - exploit/linux/local/abrt_raceabrt_priv_esc
[*] 172.16.191.170 - exploit/linux/local/abrt_sosreport_priv_esc
[*] 172.16.191.170 - exploit/linux/local/af_packet_chocobo_root_priv_esc
[*] 172.16.191.170 - exploit/linux/local/af_packet_packet_set_ring_priv_esc
[*] 172.16.191.170 - exploit/linux/local/apport_abrt_chroot_priv_esc
[*] 172.16.191.170 - exploit/linux/local/asan_suid_executable_priv_esc
[*] 172.16.191.170 - exploit/linux/local/blueman_set_dhcp_handler_dbus_priv_esc
[*] 172.16.191.170 - exploit/linux/local/bpf_priv_esc
[*] 172.16.191.170 - exploit/linux/local/bpf_sign_extension_priv_esc
[*] 172.16.191.170 - exploit/linux/local/diamorphine_rootkit_signal_priv_esc
[*] 172.16.191.170 - exploit/linux/local/docker_daemon_privilege_escalation
[*] 172.16.191.170 - exploit/linux/local/docker_privileged_container_escape
[*] 172.16.191.170 - exploit/linux/local/exim4_deliver_message_priv_esc
[*] 172.16.191.170 - exploit/linux/local/glibc_ld_audit_dso_load_priv_esc
[*] 172.16.191.170 - exploit/linux/local/glibc_origin_expansion_priv_esc
[*] 172.16.191.170 - exploit/linux/local/glibc_realpath_priv_esc
[*] 172.16.191.170 - exploit/linux/local/gnu_mailutils_maidag_url_priv_esc
[*] 172.16.191.170 - exploit/linux/local/hp_xglance_priv_esc
[*] 172.16.191.170 - exploit/linux/local/juju_run_agent_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ktsuss_suid_priv_esc
[*] 172.16.191.170 - exploit/linux/local/lastore_daemon_dbus_priv_esc
[*] 172.16.191.170 - exploit/linux/local/libuser_roothelper_priv_esc
[*] 172.16.191.170 - exploit/linux/local/nested_namespace_idmap_limit_priv_esc
[*] 172.16.191.170 - exploit/linux/local/netfilter_priv_esc_ipv4
[*] 172.16.191.170 - exploit/linux/local/network_manager_vpnc_username_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ntfs3g_priv_esc
[*] 172.16.191.170 - exploit/linux/local/omniresolve_suid_priv_esc
[*] 172.16.191.170 - exploit/linux/local/overlayfs_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ptrace_sudo_token_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ptrace_traceme_pkexec_helper
[*] 172.16.191.170 - exploit/linux/local/rds_atomic_free_op_null_pointer_deref_priv_esc
[*] 172.16.191.170 - exploit/linux/local/rds_rds_page_copy_user_priv_esc
[*] 172.16.191.170 - exploit/linux/local/recvmmsg_priv_esc
[*] 172.16.191.170 - exploit/linux/local/reptile_rootkit_reptile_cmd_priv_esc
[*] 172.16.191.170 - exploit/linux/local/servu_ftp_server_prepareinstallation_priv_esc
[*] 172.16.191.170 - exploit/linux/local/systemtap_modprobe_options_priv_esc
[*] 172.16.191.170 - exploit/linux/local/test
[*] 172.16.191.170 - exploit/linux/local/ufo_privilege_escalation
[*] 172.16.191.170 - exploit/linux/local/vmware_alsa_config
[*] 172.16.191.170 - exploit/multi/local/magnicomp_sysinfo_mcsiwrapper_priv_esc
[*] 172.16.191.170 - exploit/multi/local/xorg_x11_suid_server_modulepath
[*] 172.16.191.170 - exploit/linux/local/abrt_raceabrt_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/abrt_sosreport_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/af_packet_chocobo_root_priv_esc: The target is not exploitable. Linux kernel 4.15.0-20-generic #21-Ubuntu is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/af_packet_packet_set_ring_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/apport_abrt_chroot_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/asan_suid_executable_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/blueman_set_dhcp_handler_dbus_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/bpf_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/bpf_sign_extension_priv_esc: The target is not exploitable. Kernel version 4.15.0-20-generic is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/diamorphine_rootkit_signal_priv_esc: The target is not exploitable. Diamorphine is not installed, or incorrect signal '64'
[*] 172.16.191.170 - exploit/linux/local/docker_daemon_privilege_escalation: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/docker_privileged_container_escape: The target is not exploitable. Not inside a Docker container
[*] 172.16.191.170 - exploit/linux/local/exim4_deliver_message_priv_esc: Cannot reliably check exploitability.
[*] 172.16.191.170 - exploit/linux/local/glibc_ld_audit_dso_load_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/glibc_origin_expansion_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/glibc_realpath_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/gnu_mailutils_maidag_url_priv_esc: The target is not exploitable. /usr/local/sbin/maidag is not set-uid
[*] 172.16.191.170 - exploit/linux/local/hp_xglance_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/juju_run_agent_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/ktsuss_suid_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/lastore_daemon_dbus_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/libuser_roothelper_priv_esc: The target is not exploitable.
[+] 172.16.191.170 - exploit/linux/local/nested_namespace_idmap_limit_priv_esc: The target appears to be vulnerable.
[*] 172.16.191.170 - exploit/linux/local/netfilter_priv_esc_ipv4: The target is not exploitable.
[+] 172.16.191.170 - exploit/linux/local/network_manager_vpnc_username_priv_esc: The service is running, but could not be validated.
[*] 172.16.191.170 - exploit/linux/local/ntfs3g_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/omniresolve_suid_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/overlayfs_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/ptrace_sudo_token_priv_esc: The target is not exploitable.
[+] 172.16.191.170 - exploit/linux/local/ptrace_traceme_pkexec_helper: The target appears to be vulnerable.
[*] 172.16.191.170 - exploit/linux/local/rds_atomic_free_op_null_pointer_deref_priv_esc: The target is not exploitable. Linux kernel 4.15.0-20-generic #21-Ubuntu is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/rds_rds_page_copy_user_priv_esc: The target is not exploitable. Linux kernel version 4.15.0-20-generic is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/recvmmsg_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/reptile_rootkit_reptile_cmd_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/servu_ftp_server_prepareinstallation_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/systemtap_modprobe_options_priv_esc: The target is not exploitable.
[-] 172.16.191.170 - exploit/linux/local/test: Check failed
[*] 172.16.191.170 - exploit/linux/local/ufo_privilege_escalation: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/vmware_alsa_config: The target is not exploitable.
[*] 172.16.191.170 - exploit/multi/local/magnicomp_sysinfo_mcsiwrapper_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/multi/local/xorg_x11_suid_server_modulepath: The target is not exploitable.
[-] 172.16.191.170 - Post failed: NameError undefined local variable or method `rhost' for #<Msf::Modules::Post__Multi__Recon__Local_exploit_suggester::MetasploitModule:0x000055b7e1b1d098>
[-] 172.16.191.170 - Call stack:
[-] 172.16.191.170 -   /root/Desktop/metasploit-framework/modules/post/multi/recon/local_exploit_suggester.rb:153:in `run'
[*] Post module execution completed
msf6 post(multi/recon/local_exploit_suggester) > notes

Notes
=====

 Time  Host  Service  Port  Protocol  Type  Data
 ----  ----  -------  ----  --------  ----  ----

msf6 post(multi/recon/local_exploit_suggester) > reload
[*] Reloading module...
rmsf6 post(multi/recon/local_exploit_suggester) > run

[*] 172.16.191.170 - Collecting local exploits for x64/linux...
[*] 172.16.191.170 - The following 41 exploit checks are being tried:
[*] 172.16.191.170 - exploit/linux/local/abrt_raceabrt_priv_esc
[*] 172.16.191.170 - exploit/linux/local/abrt_sosreport_priv_esc
[*] 172.16.191.170 - exploit/linux/local/af_packet_chocobo_root_priv_esc
[*] 172.16.191.170 - exploit/linux/local/af_packet_packet_set_ring_priv_esc
[*] 172.16.191.170 - exploit/linux/local/apport_abrt_chroot_priv_esc
[*] 172.16.191.170 - exploit/linux/local/asan_suid_executable_priv_esc
[*] 172.16.191.170 - exploit/linux/local/blueman_set_dhcp_handler_dbus_priv_esc
[*] 172.16.191.170 - exploit/linux/local/bpf_priv_esc
[*] 172.16.191.170 - exploit/linux/local/bpf_sign_extension_priv_esc
[*] 172.16.191.170 - exploit/linux/local/diamorphine_rootkit_signal_priv_esc
[*] 172.16.191.170 - exploit/linux/local/docker_daemon_privilege_escalation
[*] 172.16.191.170 - exploit/linux/local/docker_privileged_container_escape
[*] 172.16.191.170 - exploit/linux/local/exim4_deliver_message_priv_esc
[*] 172.16.191.170 - exploit/linux/local/glibc_ld_audit_dso_load_priv_esc
[*] 172.16.191.170 - exploit/linux/local/glibc_origin_expansion_priv_esc
[*] 172.16.191.170 - exploit/linux/local/glibc_realpath_priv_esc
[*] 172.16.191.170 - exploit/linux/local/gnu_mailutils_maidag_url_priv_esc
[*] 172.16.191.170 - exploit/linux/local/hp_xglance_priv_esc
[*] 172.16.191.170 - exploit/linux/local/juju_run_agent_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ktsuss_suid_priv_esc
[*] 172.16.191.170 - exploit/linux/local/lastore_daemon_dbus_priv_esc
[*] 172.16.191.170 - exploit/linux/local/libuser_roothelper_priv_esc
[*] 172.16.191.170 - exploit/linux/local/nested_namespace_idmap_limit_priv_esc
[*] 172.16.191.170 - exploit/linux/local/netfilter_priv_esc_ipv4
[*] 172.16.191.170 - exploit/linux/local/network_manager_vpnc_username_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ntfs3g_priv_esc
[*] 172.16.191.170 - exploit/linux/local/omniresolve_suid_priv_esc
[*] 172.16.191.170 - exploit/linux/local/overlayfs_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ptrace_sudo_token_priv_esc
[*] 172.16.191.170 - exploit/linux/local/ptrace_traceme_pkexec_helper
[*] 172.16.191.170 - exploit/linux/local/rds_atomic_free_op_null_pointer_deref_priv_esc
[*] 172.16.191.170 - exploit/linux/local/rds_rds_page_copy_user_priv_esc
[*] 172.16.191.170 - exploit/linux/local/recvmmsg_priv_esc
[*] 172.16.191.170 - exploit/linux/local/reptile_rootkit_reptile_cmd_priv_esc
[*] 172.16.191.170 - exploit/linux/local/servu_ftp_server_prepareinstallation_priv_esc
[*] 172.16.191.170 - exploit/linux/local/systemtap_modprobe_options_priv_esc
[*] 172.16.191.170 - exploit/linux/local/test
[*] 172.16.191.170 - exploit/linux/local/ufo_privilege_escalation
[*] 172.16.191.170 - exploit/linux/local/vmware_alsa_config
[*] 172.16.191.170 - exploit/multi/local/magnicomp_sysinfo_mcsiwrapper_priv_esc
[*] 172.16.191.170 - exploit/multi/local/xorg_x11_suid_server_modulepath
[*] 172.16.191.170 - exploit/linux/local/abrt_raceabrt_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/abrt_sosreport_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/af_packet_chocobo_root_priv_esc: The target is not exploitable. Linux kernel 4.15.0-20-generic #21-Ubuntu is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/af_packet_packet_set_ring_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/apport_abrt_chroot_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/asan_suid_executable_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/blueman_set_dhcp_handler_dbus_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/bpf_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/bpf_sign_extension_priv_esc: The target is not exploitable. Kernel version 4.15.0-20-generic is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/diamorphine_rootkit_signal_priv_esc: The target is not exploitable. Diamorphine is not installed, or incorrect signal '64'
[*] 172.16.191.170 - exploit/linux/local/docker_daemon_privilege_escalation: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/docker_privileged_container_escape: The target is not exploitable. Not inside a Docker container
[*] 172.16.191.170 - exploit/linux/local/exim4_deliver_message_priv_esc: Cannot reliably check exploitability.
[*] 172.16.191.170 - exploit/linux/local/glibc_ld_audit_dso_load_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/glibc_origin_expansion_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/glibc_realpath_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/gnu_mailutils_maidag_url_priv_esc: The target is not exploitable. /usr/local/sbin/maidag is not set-uid
[*] 172.16.191.170 - exploit/linux/local/hp_xglance_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/juju_run_agent_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/ktsuss_suid_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/lastore_daemon_dbus_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/libuser_roothelper_priv_esc: The target is not exploitable.
[+] 172.16.191.170 - exploit/linux/local/nested_namespace_idmap_limit_priv_esc: The target appears to be vulnerable.
[*] 172.16.191.170 - exploit/linux/local/netfilter_priv_esc_ipv4: The target is not exploitable.
[+] 172.16.191.170 - exploit/linux/local/network_manager_vpnc_username_priv_esc: The service is running, but could not be validated.
[*] 172.16.191.170 - exploit/linux/local/ntfs3g_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/omniresolve_suid_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/overlayfs_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/ptrace_sudo_token_priv_esc: The target is not exploitable.
[+] 172.16.191.170 - exploit/linux/local/ptrace_traceme_pkexec_helper: The target appears to be vulnerable.
[*] 172.16.191.170 - exploit/linux/local/rds_atomic_free_op_null_pointer_deref_priv_esc: The target is not exploitable. Linux kernel 4.15.0-20-generic #21-Ubuntu is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/rds_rds_page_copy_user_priv_esc: The target is not exploitable. Linux kernel version 4.15.0-20-generic is not vulnerable
[*] 172.16.191.170 - exploit/linux/local/recvmmsg_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/reptile_rootkit_reptile_cmd_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/servu_ftp_server_prepareinstallation_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/systemtap_modprobe_options_priv_esc: The target is not exploitable.
[-] 172.16.191.170 - exploit/linux/local/test: Check failed
[*] 172.16.191.170 - exploit/linux/local/ufo_privilege_escalation: The target is not exploitable.
[*] 172.16.191.170 - exploit/linux/local/vmware_alsa_config: The target is not exploitable.
[*] 172.16.191.170 - exploit/multi/local/magnicomp_sysinfo_mcsiwrapper_priv_esc: The target is not exploitable.
[*] 172.16.191.170 - exploit/multi/local/xorg_x11_suid_server_modulepath: The target is not exploitable.
[*] Post module execution completed
msf6 post(multi/recon/local_exploit_suggester) > 
```

```diff
diff --git a/modules/post/multi/recon/local_exploit_suggester.rb b/modules/post/multi/recon/local_exploit_suggester.rb
index e30f232180..e0297f7c1a 100644
--- a/modules/post/multi/recon/local_exploit_suggester.rb
+++ b/modules/post/multi/recon/local_exploit_suggester.rb
@@ -149,11 +149,13 @@ class MetasploitModule < Msf::Post
       end
     end
 
+=begin
     report_note(
       :host => rhost,
       :type => 'local.suggested_exploits',
       :data => results
     )
+=end
   end
 
   def is_check_interesting?(checkcode)
```

**Comment by bcoles** [comment]
Patch #14650 is pending merge.

**Comment by adfoster-r7** [comment]
This fix should hopefully land in Metasploit 6.0.28
